cmd/ks-apiserver/app/options/options.go

```diff
@@ -42,8 +45,6 @@ import (
 	"kubesphere.io/kubesphere/pkg/simple/client/s3"
 	fakes3 "kubesphere.io/kubesphere/pkg/simple/client/s3/fake"
 	"kubesphere.io/kubesphere/pkg/simple/client/sonarqube"
```

Comment:
Some of the files have four parts. I used to merge them into three parts manually.

Reply:
I think we need a format guideline which all developers can follow, also it's useful when setting up CI.